_resources/port1.0/fetch/mirror_sites.tcl

```diff
@@ -14,242 +14,204 @@ set portfetch::mirror_sites::sites(afterstep) {
     ftp://ftp.kddlabs.co.jp/X11/AfterStep/
 }
 
+# The link: http://www.apache.org/dyn/closer.cgi
+# Maybe that one's enough, and we'll rely on their “cdn”
 set portfetch::mirror_sites::sites(apache) {
-    https://mirror.aarnet.edu.au/pub/apache/
+    http://www.apache.org/dyn/closer.cgi
+    http://apache.mirrors.spacedump.net
+    https://downloads.apache.org
```

Comment:
We should not include this URL because of you visit this URL it says in bold:

> **Please do not download releases from downloads.apache.org unless you absolutely have to!**